const-oid/oiddbgen/Cargo.toml

```diff
@@ -8,7 +8,7 @@ convert_case = "0.5.0"
 proc-macro2 = "1.0.36"
 quote = "1.0.15"
 regex = "1.5.5"
-syn = "1.0.86"
+syn = "2.0.6"
```

Comment:
Is this the minimum compatible version?

I ask because if it's actually `2` I'd prefer to use that instead. We run `minimum-version` checks, and including an explicit minor version like that triggers Dependabot runs.

Reply:
2 works fine, I've relaxed it to 2.